cartoframes/context.py

```diff
@@ -184,6 +190,53 @@ def _table_exists(self, table_name):
 
         return False
 
+    def _send_batches(self, df, table_name, temp_dir, geom_col, lnglat):
+        """Batch sending a dataframe"""
+        subtables = []
+        # send dataframe chunks to carto
+        for chunk_num, df_chunk in tqdm(df.groupby([i // MAX_IMPORT_ROWS
```

Comment:
I wish I didn't have to create a list this large, and also wish there was a generator version of the `df.groupyby` so the amount of memory being used would be reduced by `len(df) / num_batches`